client/src/components/ManageAccount/tests/ManageAccountTables.test.jsx

```diff
@@ -154,7 +154,33 @@ describe('Third Party Apps', () => {
     );
     expect(getByText(/3rd Party Apps/)).toBeDefined();
     expect(getByText('Google Drive')).toBeDefined();
-    expect(getByText('Disconnect')).toBeDefined();
```

Comment:
Is this a desired change?

Reply:
The third party test is a desired change on my part. Just in case if there is other systems added in the future other than the Google Drive integration in the account setup section. It also covers for a code coverage case where google drive is disabled. As for queryByText this has been reverted in the most recent commit.

Reply:
Good additions 👍 . Specifically though, is `expect(getByText('Disconnect')).toBeDefined();` intended to be removed?